backend/api/views.py

```diff
@@ -1,3 +1,25 @@
-from django.shortcuts import render
+# from django.shortcuts import render
+from django.http import HttpResponse
 
-# Create your views here.
+from django.views.decorators.csrf import csrf_exempt
+from .validators import SchemaValidator
+import json
+
+
+schema = SchemaValidator()
```

Comment:
Is not `validator = SchemaValidator()` better ?

(and `validator.validate(json_data)`)

Reply:
indeed. that was my first approach. I don't know why I changed
ty